homeassistant/components/sensor/tellduslive.py

```diff
@@ -11,26 +11,32 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-SENSOR_TYPE_TEMP = 'temp'
+SENSOR_TYPE_TEMPERATURE = 'temp'
 SENSOR_TYPE_HUMIDITY = 'humidity'
 SENSOR_TYPE_RAINRATE = 'rrate'
 SENSOR_TYPE_RAINTOTAL = 'rtot'
 SENSOR_TYPE_WINDDIRECTION = 'wdir'
 SENSOR_TYPE_WINDAVERAGE = 'wavg'
 SENSOR_TYPE_WINDGUST = 'wgust'
+SENSOR_TYPE_UV = 'uv'
 SENSOR_TYPE_WATT = 'watt'
 SENSOR_TYPE_LUMINANCE = 'lum'
+SENSOR_TYPE_DEW_POINT = 'dewp'
+SENSOR_TYPE_BAROMETRIC_PRESSURE = 'barpress'
 
 SENSOR_TYPES = {
-    SENSOR_TYPE_TEMP: ['Temperature', TEMP_CELSIUS, 'mdi:thermometer'],
+    SENSOR_TYPE_TEMPERATURE: ['Temperature', TEMP_CELSIUS, 'mdi:thermometer'],
     SENSOR_TYPE_HUMIDITY: ['Humidity', '%', 'mdi:water'],
-    SENSOR_TYPE_RAINRATE: ['Rain rate', 'mm', 'mdi:water'],
+    SENSOR_TYPE_RAINRATE: ['Rain rate', 'mm/h', 'mdi:water'],
     SENSOR_TYPE_RAINTOTAL: ['Rain total', 'mm', 'mdi:water'],
     SENSOR_TYPE_WINDDIRECTION: ['Wind direction', '', ''],
     SENSOR_TYPE_WINDAVERAGE: ['Wind average', 'm/s', ''],
     SENSOR_TYPE_WINDGUST: ['Wind gust', 'm/s', ''],
+    SENSOR_TYPE_UV: ['UV', 'UV', ''],
     SENSOR_TYPE_WATT: ['Watt', 'W', ''],
```

Comment:
While you're at it, can you please change this to `Power` instead of `Watt`, which is the proper physical quantity? (Watt is still the unit)

Reply:
You're right, changed it now